### PR TITLE
Fix bazinga usage

### PR DESCRIPTION
**Pull request by adriendupuis**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | N/A
| Versions      | master, 4.6, 3.3
| Edition       | All

`bazinga` must generate the translations in the `./public/assets` directory, not the `./assets` directory.
It also means that removing `./assets` doesn't remove translations.

`composer run post-update-cmd` is clearing the cache, and is running `bazinga` and `yarn`. It's the best way to get the translations working. As it clears only the default SiteAccess cache, the Back Office SiteAccess cache may need to be cleared as well.

See `composer.json` script using `bazinga:js-translation:dump %PUBLIC_DIR%/assets --merge-domains` in [4.6's `manifest.json`](https://github.com/ibexa/recipes/blob/master/ibexa/oss/4.6/manifest.json#L95), or [3.3.15 update doc](https://doc.ibexa.co/en/latest/update_and_migration/from_3.3/update_from_3.3/#v3315), or [4.0 update doc](https://doc.ibexa.co/en/latest/update_and_migration/from_3.3/to_4.0/#finish-update).

Previews:
- About assets:
    - https://doc.ibexa.co/en/latest/administration/project_organization/project_organization/#assets has a "Removing `assets` manually" foldable box. (side note, this box state is inherited from [3.0 moving to Symfony 4](https://github.com/ibexa/documentation-developer/pull/647/files#diff-97895baf8cabade76e43d75c65d9d0c96f64295f5a7e1ad8aec6847a0627fe00R38-R53)) 
    - https://ez-systems-developer-documentation--2500.com.readthedocs.build/en/2500/administration/project_organization/project_organization/#assets doesn't have this box.
- About cache clearing:
    - https://ez-systems-developer-documentation--2500.com.readthedocs.build/en/2500/multisite/languages/back_office_translations/#enabling-back-office-languages _Enabling Back Office languages_ "Tip" box.
    - https://ez-systems-developer-documentation--2500.com.readthedocs.build/en/2500/multisite/languages/back_office_translations/#custom-string-translations _Custom string translations_ "Note" box at page bottom.

Nota bene: This PR focus on `bazinga`. https://ez-systems-developer-documentation--2500.com.readthedocs.build/en/2500/administration/project_organization/project_organization/#assets remaining part sounds wrong as Encore is supposed to be used, and "for example assets/js/script.js can be referenced as js/script.js from templates" doesn't make any sens. It's kept for another time and another PR about assets.

#### Checklist

- [ ] Text renders correctly
- [ ] Text has been checked with vale
- [ ] Description metadata is up to date
- [ ] Redirects cover removed/moved pages
- [ ] Code samples are working
- [ ] PHP code samples have been fixed with PHP CS fixer
- [ ] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

File: docs/administration/project_organization/project_organization.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ All project assets are accessible through the `assets` path.
     the `yarn encore <dev|prod>` command:
     
     ```
-    php bin/console bazinga:js-translation:dump assets --merge-domains
+    php bin/console bazinga:js-translation:dump public/assets --merge-domains
```
Comment (adriendupuis) [inline docs/administration/project_organization/project_organization.md:37]: So it's not related to ./assets anymore and shouldn't be mentioned as a note…

File: docs/multisite/languages/back_office_translations.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ If your browser language is set to French, the Back Office will be displayed in
 
     `ibexa.system.<siteaccess>.user_preferences.additional_translations: ['pl_PL', 'fr_FR']`
 
-    Then, run `composer run post-update-cmd` and clear the cache.
+    Then, run `composer run post-update-cmd` which include a cache clear.
```
Comment (adriendupuis) [inline docs/multisite/languages/back_office_translations.md:33]: Mayne cache:clear for other SiteAccesses than the default one?

File: docs/multisite/languages/back_office_translations.md [hunk 3]
```diff
@@ -120,4 +120,4 @@ The language to display will then be selected automatically based on [user prefe
 
 !!! note
 
-    You may need to clear the cache after adding new translations, before they are available in the Back Office.
+    Run `composer run post-update-cmd` which install your translations to JavaScript through Bazinga, and also clear the cache.
```
Comment (adriendupuis) [inline docs/multisite/languages/back_office_translations.md:123]:
```suggestion
    Run `composer run post-update-cmd` which both install your translations to JavaScript through Bazinga, and clear the default SiteAccess cache.
    Run `php bin/console cache:clear --siteaccess=admin` to clear the Back Office cache (adapt SiteAccess name if needed).
```

File: docs/multisite/languages/back_office_translations.md [hunk 4]
```diff
@@ -120,4 +120,5 @@ The language to display will then be selected automatically based on [user prefe
 
 !!! note
 
-    You may need to clear the cache after adding new translations, before they are available in the Back Office.
+    Run `composer run post-update-cmd` which both install your translations to JavaScript through Bazinga, and clear the default SiteAccess cache.
+    Run `php bin/console cache:clear --siteaccess=admin` to clear the Back Office cache (adapt SiteAccess name if needed).
```
Comment (adriendupuis) [inline docs/multisite/languages/back_office_translations.md:124]:
```suggestion
    Run `php bin/console cache:clear --siteaccess=admin` to clear the Back Office cache (adapt SiteAccess if needed).
```

**Review by lionelakpagni — APPROVED** [review]

File: docs/multisite/languages/back_office_translations.md [hunk 5]
```diff
@@ -117,4 +117,5 @@ The language to display is then selected automatically based on [user preference
 
 !!! note
 
-    You may need to clear the cache after adding new translations, before they're available in the back office.
+    Run `composer run post-update-cmd` which both install your translations to JavaScript through Bazinga, and clear the default SiteAccess cache.
```
Comment (mikadamczyk) [inline docs/multisite/languages/back_office_translations.md:120]: If we want to mention the Bazinga  I would go with the full name `BazingaJsTranslationBundle`

**Review by mikadamczyk — APPROVED** [review]

File: docs/multisite/languages/back_office_translations.md (hunk 5 above)
Comment (adriendupuis) [reply docs/multisite/languages/back_office_translations.md:120]:
```suggestion
    Run `composer run post-update-cmd` which both install your translations to JavaScript using the `BazingaJsTranslationBundle`, and clear the default SiteAccess cache.
```

File: docs/multisite/languages/back_office_translations.md [hunk 6]
```diff
@@ -117,4 +117,5 @@ The language to display is then selected automatically based on [user preference
 
 !!! note
 
-    You may need to clear the cache after adding new translations, before they're available in the back office.
+    Run `composer run post-update-cmd` which both install your JavaScript translations using the `BazingaJsTranslationBundle`, and clear the default SiteAccess cache.
+    Run `php bin/console cache:clear --siteaccess=admin` to clear the Back Office cache (adapt SiteAccess if needed).
```
Comment (mnocon) [inline docs/multisite/languages/back_office_translations.md:121]:
```suggestion
    Run `php bin/console cache:clear --siteaccess=admin` to clear the back office cache (adapt SiteAccess if needed).
```
I thought we had a rule in vale for that 🤔

Comment (mnocon) [reply docs/multisite/languages/back_office_translations.md]: Ok, I think it just generates a warning: https://github.com/ibexa/vale-styles/blob/main/styles/Ibexa/DontCapitalize.yml#L12

**Review by mnocon — APPROVED** [review]

File: docs/multisite/languages/back_office_translations.md [hunk 7]
```diff
@@ -117,4 +117,5 @@ The language to display is then selected automatically based on [user preference
 
 !!! note
 
-    You may need to clear the cache after adding new translations, before they're available in the back office.
+    Run `composer run post-update-cmd` which installs your JavaScript translations by using the `BazingaJsTranslationBundle`, and clears the default SiteAccess cache.
```
Comment (adriendupuis) [inline docs/multisite/languages/back_office_translations.md:120]:
```suggestion
    Run `composer run post-update-cmd` which installs your JavaScript translations by using `BazingaJsTranslationBundle`, and clears the default SiteAccess cache.
```

File: docs/multisite/languages/back_office_translations.md [hunk 8]
```diff
@@ -117,4 +117,5 @@ The language to display is then selected automatically based on [user preference
 
 !!! note
 
-    You may need to clear the cache after adding new translations, before they're available in the back office.
+    Run `composer run post-update-cmd` which installs your JavaScript translations by using `BazingaJsTranslationBundle`, and clears the default SiteAccess cache.
```
Comment (adriendupuis) [inline docs/multisite/languages/back_office_translations.md:120]:
```suggestion
    Run `composer run post-update-cmd` which installs your JavaScript translations by using `BazingaJsTranslationBundle`, and clears the cache of the default Siteaccess.
```

File: docs/multisite/languages/back_office_translations.md [hunk 9]
```diff
@@ -117,4 +117,5 @@ The language to display is then selected automatically based on [user preference
 
 !!! note
 
-    You may need to clear the cache after adding new translations, before they're available in the back office.
+    Run `composer run post-update-cmd` which installs your JavaScript translations by using `BazingaJsTranslationBundle`, and clears the cache of the default Siteaccess.
+    Run `php bin/console cache:clear --siteaccess=admin` to clear the back office cache (adapt SiteAccess if needed).
```
Comment (adriendupuis) [inline docs/multisite/languages/back_office_translations.md:121]:
```suggestion
    Run `composer run post-update-cmd` which installs your JavaScript translations by using `BazingaJsTranslationBundle`,
    and clears the cache of the default Siteaccess.

    Run `php bin/console cache:clear --siteaccess=admin` to clear the back office cache.
    You may need to replace `admin` with the back office's SiteAccess name used in your installation.
```

**Review by dabrt — APPROVED** [review]

File: docs/multisite/languages/back_office_translations.md [hunk 10]
```diff
@@ -117,4 +117,8 @@ The language to display is then selected automatically based on [user preference
 
 !!! note
 
-    You may need to clear the cache after adding new translations, before they're available in the back office.
+    Run `composer run post-update-cmd` which installs your JavaScript translations by using `BazingaJsTranslationBundle`,
+    and clears the cache of the default Siteaccess.
```
Comment (adriendupuis) [inline docs/multisite/languages/back_office_translations.md:121]:
```suggestion
    and clears the cache of the default SiteAccess.
```

**Pull request merged**